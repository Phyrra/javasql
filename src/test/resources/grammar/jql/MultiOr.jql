field1 = 1 or field2 = 2 or field3 = 3