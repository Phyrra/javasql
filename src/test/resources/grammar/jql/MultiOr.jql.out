(([field1] = 1 OR [field2] = 2) OR [field3] = 3)